lib/sql-string.js

```diff
@@ -35,6 +35,11 @@ function escape(val, timeZone, dialect, format) {
     break;
   }
 
+  // moment
+  if (val.toDate) {
```

Comment:
Passing `format` output is already supported by sequelize but passing `moment` object isn't.

With this PR we will be able to support it. But we need to check for `isMoment` or `instanceof moment` rather than `if( val.toDate )`